app/views/contexts/section_summary_context.py

```diff
@@ -159,7 +159,9 @@ def _list_summary_element(self, summary, current_location, section) -> Mapping:
             "type": rendered_summary["type"],
             "add_link": add_link,
             "add_link_text": rendered_summary["add_link_text"],
-            "empty_list_text": rendered_summary["empty_list_text"],
+            "empty_list_text": rendered_summary["empty_list_text"]
+            if "empty_list_text" in summary.keys()
+            else "",
```

Comment:
This could be simpler with a `get`:
```suggestion
            "empty_list_text": rendered_summary.get("empty_list_text")
```

It also shouldn't be setting an empty string, as that will mean the template will output an empty `div`. The template should be amended to add a check for `empty_list_text`. So in `sectionsummary.html`:

```
            {% include 'partials/summary/list-summary.html' %}
            {% else %}
              <div class="u-mb-s">
                {{ summary.empty_list_text }}
              </div>
            {% endif %}
```

will become:
```
            {% include 'partials/summary/list-summary.html' %}
            {% elif summary.empty_list_text %}
              <div class="u-mb-s">
                {{ summary.empty_list_text }}
              </div>
            {% endif %}
```

Reply:
Added and done! Thanks👍